http-filter-example/http_filter.cc

```diff
@@ -36,15 +36,15 @@ HttpSampleDecoderFilter::HttpSampleDecoderFilter(HttpSampleDecoderFilterConfigSh
     auto tokens = StringUtil::splitToken(operation, " ");
     if (tokens.size() < 3) {
       fail("too few arguments provided");
-      setError(1); // TODO: set error based on globally defined macros
-      return;
+      setError();
+      return; // TODO: should we quit here or continue to process operations that are grammatical?
```

Comment:
I think we can stop processing if any of the config is unparse-able at init time